plugins/gwapi/commands/check.md

```
@@ -0,0 +1,147 @@
+---
+description: Check Gateway API resources status in the cluster
+argument-hint: [namespace]
+---
+
+## Name
+gwapi:check
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)